frontend/pages/SoftwarePage/SoftwareOSDetailsPage/_styles.scss

```diff
@@ -4,16 +4,7 @@
   flex-direction: column;
   gap: $pad-medium;
 
-  &__vulnerabilities-section {
-    background-color: $core-white;
-    padding: $pad-xxlarge;
-    border: 1px solid $ui-fleet-black-10;
-    border-radius: $border-radius-xxlarge;
-    box-shadow: $box-shadow;
-
-    h2 {
-      margin: 0;
-      font-size: $medium;
-    }
+  h2 {
+    font-size: $small;
```

Comment:
@rachaelshaw The Figma shows these card header text as 18px, but in `frontend/styles/var/fonts.scss` we only have `$small` as 16px and `$medium` as 20px. Is the intended header text 18px? If so, I'm going to make a `$smedium` text size?

Reply:
@rachaelshaw @marko-lisica @mike-j-thomas something i've been thinking is that it would be nice to have reusable header components in the style guide that we use throughout the app. We could then build this header component to match the style guide and should always have consistent headers.